src/index.html

```diff
@@ -3824,4 +3824,4 @@ <h4>Indicator chart title</h4>
 }
 </style>
 
-</body></html>
+<script src="js/index.js"></script></body></html>
```

Comment:
Doesn't this get added by the import script now? - The import script included index.html right?

Reply:
Ah, yes it does, which is why it's there